Makefile

```diff
@@ -0,0 +1,7 @@
+.PHONY: docker-clean
+docker-clean:
+	@docker compose down -v --remove-orphans
+
+.PHONY: docker-test
+docker-test:
+	@docker compose build test && docker compose run --rm test go test -race=1 -v ./...
```

Comment:
Maybe add `GODEBUG=cgocheck=2` for cgo

Reply:
Are you going to add an action to run tests?

Reply:
We can do it, but i need to setup CI access to rocksdb repo then

Reply:
We can do it separately.